Sources/ScipioKit/Producer/Cache/CacheSystem.swift

```diff
@@ -247,7 +245,10 @@ struct CacheSystem: Sendable {
             buildOptions: buildOptions,
             clangVersion: clangVersion,
             xcodeVersion: xcodeVersion,
-            scipioVersion: currentScipioVersion
+            // Making the cache key compatible with 0.24.0 temporarily for easier debugging.
+            //
+            // TODO: revert this before merging
+            scipioVersion: "578ce98d236e79dad3e473cb11153e867be07174"
```

Comment:
Disabling when developing is not a solution, because excluding `scipioVersion` from cache keys, those are not compatible with the cache keys generated by release-versioned Scipio.

Ideally we should introduce some Scipio's cache versioning and increment it when there are some incompatible changes to built frameworks (but it'll make Scipio's development and review process a bit complicated).